### PR TITLE
Change examples to use WebApplication V1

### PR DESCRIPTION
**Pull request by JamesNK**
There are a lot of examples. Splitting this up over multiple PRs.

### CODE REVIEW

File: examples/Blazor/Server/Program.cs [hunk 1]
```diff
@@ -16,23 +16,35 @@
...
+    opts.MimeTypes = ResponseCompressionDefaults.MimeTypes.Concat(
+        new[] { "application/octet-stream" });
+});
+
+var app = builder.Build();
+app.UseResponseCompression();
+
+if (builder.Environment.IsDevelopment())
+{
+    app.UseDeveloperExceptionPage();
```
Comment (JamesNK) [inline examples/Blazor/Server/Program.cs:35]: Is `UseDeveloperExceptionPage` needed with `WebApplication`?

Comment (captainsafia) [reply examples/Blazor/Server/Program.cs:35]: Nope, it's automatically added when the environment is a development environment.

File: examples/Aggregator/Server/Program.cs [hunk 2]
```diff
@@ -16,23 +16,50 @@
...
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using Server;
 
-namespace Server
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddGrpc();
+builder.Services.AddSingleton<IncrementingCounter>();
+
+if (bool.TryParse(builder.Configuration["EnableOpenTelemetry"], out var enableOpenTelemetry) && enableOpenTelemetry)
```
Comment (captainsafia) [inline examples/Aggregator/Server/Program.cs:29]: Is there no constant for the `EnableOpenTelemetry` key name?

File: examples/Blazor/Server/Program.cs [hunk 3]
```diff
@@ -16,23 +16,35 @@
...
+    app.UseWebAssemblyDebugging();
 }
+
+app.UseStaticFiles();
+app.UseBlazorFrameworkFiles();
+
+app.UseRouting();
```
Comment (captainsafia) [inline examples/Blazor/Server/Program.cs:42]: `UseRouting` is added automatically.

File: examples/Blazor/Server/Server.csproj [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net7.0</TargetFramework>
+    <UserSecretsId>46982f83-f153-443e-b589-4b2bc7b5945e</UserSecretsId>
```
Comment (captainsafia) [inline examples/Blazor/Server/Server.csproj:5]: What is the user secret needed for?

File: examples/Certifier/Client/Program.cs [hunk 5]
```diff
@@ -71,6 +71,8 @@ private static async Task CallCertificateInfo(bool includeClientCertificate)
                 Console.WriteLine($"Unexpected error calling service.");
                 throw;
             }
+
+            Console.WriteLine();
```
Comment (captainsafia) [inline examples/Certifier/Client/Program.cs:75]: Is this intentional?

Comment (JamesNK) [reply examples/Certifier/Client/Program.cs:75]: Yes

File: examples/Blazor/Server/Server.csproj (hunk 4 above)
Comment (JamesNK) [reply examples/Blazor/Server/Server.csproj:5]: VS adds it automatically 🤷

File: examples/Blazor/Server/Program.cs (hunk 3 above)
Comment (JamesNK) [reply examples/Blazor/Server/Program.cs:42]: Appears to be required here for Blazor (and it's present in Blazor template)

File: examples/Aggregator/Server/Program.cs (hunk 2 above)
Comment (JamesNK) [reply examples/Aggregator/Server/Program.cs:29]: It's only used once.

**Comment by JamesNK** [comment]
Feedback applied.

**Review by captainsafia — APPROVED** [review]

**Pull request merged**